tests/bwc/test_rolling_upgrade.py

```diff
@@ -41,7 +41,8 @@
     UpgradePath('5.8.x', '5.9.x'),
     UpgradePath('5.9.x', '5.10.x'),
     UpgradePath('5.10.x', '5.10'),
-    UpgradePath('5.10', 'latest-nightly'),
+    UpgradePath('5.10', '6.0.x'),
```

Comment:
- Should we add another step and test 6.0 branch?

Reply:
I'd tend for the other way, remove the `5.10` branch and upgrade from `5.10.x` -> `6.0.x`.
I do not see any reason to include concrete branches, I think we just did because of the problem that `5.10.0` -> `6.x` didn't work.

Reply:
yep, I remember that, just thought it could be helpful, especially until 6.0 becomes stable, but anyway we can always add it in the future if there is need, thx!

Reply:
Updated, (force pushed) and removed the 5.10 branch, please check again, thx!